src/test/scala/org/broadinstitute/dsde/workbench/sam/service/UserServiceSpecs/SupportSummarySpec.scala

```diff
@@ -110,24 +124,20 @@ class SupportSummarySpec extends UserServiceTestTraits with TimeMatchers {
         TermsOfServiceDetails(Option("v1"), Option(nowInstant), permitsSystemUsage = true, isCurrentVersion = true),
         GroupMembershipCounts(7, 42, 101),
         Map("enterpriseFeatures" -> FilteredResourcesFlat(Set(enterpriseFeature)).toJson),
-        favoriteResources
+        favoriteResources,
+        Option(
+          List(
+            GroupMembershipCount(testGroupName, 1),
+            GroupMembershipCount(testPolicyId, 2)
+          )
+        )
       )
 
       // Act
-      val response = runAndWait(userService.getSamUserCombinedState(testUser, samRequestContext, mockResourceService))
+      val response = runAndWait(userService.getSamUserCombinedState(testUser, 1, samRequestContext, mockResourceService))
 
       // Assert
-      response.samUser should be(testUser)
-      response.allowances should be(expected.allowances)
-      response.attributes should be(expected.attributes)
-      response.termsOfServiceDetails.acceptedOn.get should be(expected.termsOfServiceDetails.acceptedOn.get)
-      response.termsOfServiceDetails.isCurrentVersion should be(expected.termsOfServiceDetails.isCurrentVersion)
-      response.termsOfServiceDetails.permitsSystemUsage should be(expected.termsOfServiceDetails.permitsSystemUsage)
-      response.termsOfServiceDetails.latestAcceptedVersion should be(expected.termsOfServiceDetails.latestAcceptedVersion)
-      response.additionalDetails should be(Map("enterpriseFeatures" -> filteredResourcesFlat.toJson))
-      response.favoriteResources should be(favoriteResources)
-      response.groupMembershipCounts.directSynchronized should be(7)
-      response.groupMembershipCounts.totalSynchronized should be(42)
+      response should be(expected)
```

Comment:
this is a deep comparison, no need to check each field separately. Then when we add new fields (like in this PR), the check comes for free

Reply:
thank you, I wondered about this when I was working in this area too